physical/raft/snapshot_test.go

```diff
@@ -958,3 +961,48 @@ func TestBoltSnapshotStore_CloseFailure(t *testing.T) {
 		t.Fatal("expected write to fail")
 	}
 }
+
+// TestLoadSnapshot loads a test snapshot file and verifies that there are no
+// errors, and that the expected paths are excluded from the FSM.
+func TestLoadReadOnlySnapshot(t *testing.T) {
```

Comment:
nit: update so godoc and function name match

Reply:
Fixed in 3c1c8cb99d6779af9c8e127c99dfa9dec3974492